changes/tasks.py

```diff
@@ -1860,6 +1860,7 @@ def get_engage_inbound_and_reply(wa_contact_id, message_id):
     inbound_text = " | ".join(list(inbound_text)[::-1])
     labels = map(lambda m: m["_vnd"]["v1"]["labels"], inbounds)
     labels = map(lambda l: l["value"], ichain.from_iterable(labels))
+    message_id = message_id
```

Comment:
This doesn't really do anything, so we can probably remove this line.

Reply:
noted